build.py

```diff
@@ -165,10 +168,18 @@ def check_mlx():
 
 def generate_extra_mk():
     global extra_libs
+    global cxx_flags
+    global ld_flags
 
     with open('core/extra.mk', 'w') as fp:
-        fp.write('LIBS += %s ' % \
+        fp.write('LIBS += %s\n' % \
                 ' '.join(map(lambda lib: '-l' + lib, extra_libs)))
+        fp.write('CFLAGS += %s\n' % \
```

Comment:
Maybe we can skip this one, since i) `CFLAGS` is never used and ii) in general there is no guarantee that flags for C++ will also work for C.